src/timestamps-mapper/TimestampToBlockNumberMapper.sol

```diff
@@ -141,21 +141,22 @@ contract TimestampToBlockNumberMapper {
         uint256 currentElement = remappedBlocksAmount / 2;
```

Comment:
We have to keep track of left and right bound of current interval and not just the middle element. So we have to declare `leftBound = 0` and `rightBound = remappedBlocksAmount`